internal/backoff/exponential.go

```diff
@@ -34,37 +35,47 @@ type ExponentialOption func(*exponentialOptions)
 
 // exponentialOptions are the configuration options for an exponential backoff
 type exponentialOptions struct {
-	base, min, max time.Duration
-	rand           *rand.Rand
-	minMaxDiff     int64
+	first, max time.Duration
+	newRand    func() *rand.Rand
 }
 
 func (e exponentialOptions) validate() (err error) {
-	if e.base <= 0 {
-		err = multierr.Append(err, errors.New("invalid base for exponential backoff, need greater than zero"))
-	}
-	if e.min < 0 {
-		err = multierr.Append(err, errors.New("invalid min for exponential backoff, need greater than or equal to zero"))
+	if e.first <= 0 {
+		err = multierr.Append(err, errors.New("invalid first duration for exponential backoff, need greater than zero"))
 	}
 	if e.max < 0 {
 		err = multierr.Append(err, errors.New("invalid max for exponential backoff, need greater than or equal to zero"))
 	}
-	if e.max < e.min {
-		err = multierr.Append(err, errors.New("exponential max value must be greater than min value"))
-	}
 	return err
 }
 
+func newRand() *rand.Rand {
+	return rand.New(rand.NewSource(time.Now().UnixNano()))
+}
+
 var defaultExponentialOpts = exponentialOptions{
-	base: time.Millisecond,
-	max:  time.Hour, // :shrug:
-	rand: rand.New(rand.NewSource(time.Now().UnixNano())),
+	first:   100 * time.Millisecond,
```

Comment:
Can we make `defaultExponentialOpts` a public variable so this can be seen in documentation?  We should also talk about all the arbitrary default durations around the codebase.